mir-module/src/main/js/src/editor/funding/openaire.ts

```diff
@@ -0,0 +1,182 @@
+import { Award, Funder, Funding } from './types';
+
+type OpenAIREMetadata = {
+  fundingtree?:
+    | { funder: { id: { $: string }; name: { $: string } } }[]
+    | { funder: { id: { $: string }; name: { $: string } } };
+  title?: { $: string };
+  code?: { $: string };
+};
+
+type OpenAIREAPIResponseItem = {
+  metadata: {
+    'oaf:entity': {
+      'oaf:project': OpenAIREMetadata;
+    };
+  };
+};
+
+// enum is currently limited to more famous funders
+export enum OpenAIREFunderId {
+  EC = 'ec__________::EC',
+  NIH = 'nih_________::NIH',
+  UKRI = 'ukri________::UKRI',
+  NSF = 'nsf_________::NSF',
+  FCT = 'fct_________::FCT',
+  WT = 'wt__________::WT',
+  ANR = 'anr_________::ANR',
+  ARC = 'arc_________::ARC',
+  NHMRC = 'nhmrc_______::NHMRC',
+  NWO = 'nwo_________::NWO',
+  FWF = 'fwf_________::FWF',
+  TUBITAK = 'tubitakf____::tubitak',
+  AKA = 'aka_________::AKA',
+  SFI = 'sfi_________::SFI',
+  SNSF = 'snsf________::SNSF',
+  MZOS = 'irb_hr______::MZOS',
```

Comment:
this looks wrong?

Reply:
That's what i thought at first too, but it's actually correct. See http://api.openaire.eu/search/projects?funder=MZOS&format=json for examples.